webui/ocw/lib/gce.py

```diff
@@ -30,35 +30,75 @@ def compute_client(self):
             self.__compute_clinet = googleapiclient.discovery.build('compute', 'v1', credentials=credentials)
         return self.__compute_clinet
 
-    def list_instances(self, zone='europe-west1-b'):
-        i = self.compute_client().instances().list(project=self.__project, zone=zone).execute()
-        return i['items'] if 'items' in i else []
+    def list_instances(self, zone):
```

Comment:
what method is doing looks strange and requires some explanations ( even for yourself are sure that you won't  forget what's happening here in 6 months ? )